cmd/livepeer/livepeer.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/livepeer/go-livepeer/drivers"
 	"github.com/livepeer/go-livepeer/eth"
 	"github.com/livepeer/go-livepeer/eth/eventservices"
+
```

Comment:
stray?

Reply:
Actually, that's a space automatically added by Visual Code. Must be because of the commented out import